tests/test_rotate.py

```diff
@@ -123,7 +123,7 @@ def test_trig_invariance(angle: float, n: int):
     assert isclose(r_sin, n_sin, rel_to=[n / 1e9])
 
 
-@given(v=vectors(), angle=angles(), n=st.integers(min_value=0, max_value=1e5))
+@given(v=vectors(), angle=angles(), n=st.integers(min_value=0, max_value=10_000))
```

Comment:
10,000 is 1e4. Was this change intentional?

Reply:
Oops, no; let me PR this back, thanks for the catch.